desktop/models/search_result.coffee

```diff
@@ -30,7 +30,7 @@ module.exports = class SearchResult extends Backbone.Model
     _s.trim(_s.truncate(@get('display'), 75))
 
   location: ->
-    if @get('model') is 'profile'
+    if @get('model') is 'profile' || @get('model') is 'page'
       "/#{@get('id')}"
```

Comment:
we assume the page has a redirect in the root path for its slug, as with `/consign` and `/collector-loyalty`.